src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AcceptanceTestsInSeparateSubprojectFunctionalSpec.groovy

```diff
@@ -16,4 +18,34 @@ class AcceptanceTestsInSeparateSubprojectFunctionalSpec extends AbstractPitestFu
             result.getStandardOutput().contains('Generated 4 mutations Killed 3 (75%)')
     }
 
+    void "should aggregate report from subproject"() {
```

Comment:
This test started to fail after changed. I checked it and it seems you could hit some upstream aggregation problem, so I reworked tests to workaround that problem (could be fixed later on). I pushed changed to your fork. After that the test started failing with:

```
org.gradle.api.GradleException: Build aborted because of an internal error.

	at nebula.test.functional.internal.DefaultExecutionResult.rethrowFailure(DefaultExecutionResult.groovy:97)
	at nebula.test.Integration$Trait$Helper.runTasksSuccessfully(Integration.groovy:150)
	at info.solidsoft.gradle.pitest.functional.AcceptanceTestsInSeparateSubprojectFunctionalSpec.should aggregate report from subproject(AcceptanceTestsInSeparateSubprojectFunctionalSpec.groovy:31)
Caused by: org.gradle.internal.exceptions.LocationAwareException: Execution failed for task ':pitestReportAggregate'.
...
Caused by: org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':pitestReportAggregate'.
...
Caused by: java.lang.IllegalStateException: Failed to build: no lineCoverageFiles have been set
	at org.pitest.aggregate.ReportAggregator$Builder.validateState(ReportAggregator.java:226)
	at org.pitest.aggregate.ReportAggregator$Builder.build(ReportAggregator.java:214)
	at info.solidsoft.gradle.pitest.report.PitestReportAggregator.aggregate(PitestReportAggregator.groovy:38)
	at info.solidsoft.gradle.pitest.AggregateReportTask.aggregate(AggregateReportTask.groovy:16)
...

Maybe you just don't set it in the upstream `ReportAggregator`?

Reply:
Thanks. Test fixed. 
Ive merge your commit into my